README.md

```diff
@@ -783,11 +783,14 @@ available.
 This is known to work on a Linux development host system. The user
 must be allowed to use Docker.
 
-KVM must be enabled and the user must be allowed to use it. Usually this
-is done by adding the user to the `kvm` group. The
-["Install QEMU-KVM"](https://clearlinux.org/documentation/clear-linux/get-started/virtual-machine-install/kvm)
-section in the Clear Linux documentation contains further information
-about enabling KVM.
+KVM must be enabled. Usually this is the case when `/dev/kvm` exists.
+The current user does not need the privileges to use KVM and QEMU
```

Comment:
Nitpick question, do we need to decide to use **does not** or **doesn't** and stick to it trough the `md` documents? or does not matter that much? 
It came to my attention because in this line **does not** is used and the next one is **doesn't**
(And I think this will apply for **do not** and **don't** too and all the shorter version of ENG words)